update-motd.d/99-livepatch

```diff
@@ -54,8 +54,8 @@ print_status() {
 }
 
 
-if [[ ":$PATH:" != *":/snap/bin:"* ]]; then
-    PATH=:/snap/bin:$PATH
+if ! echo "$PATH" | grep -qE "/snap/bin(:|$|/:/$)"; then
```

Comment:
I think the regexp you want here is: `grep -E "(^|:)/snap/bin/?(:|$)"`

The initial group is needed to avoid matching `/foo/snap/bin`, and at the end you want either a colon or the end of the string, after the optional final `/`.